libs/crosstarget-utils/src/wasm/regex.rs

```diff
@@ -0,0 +1,39 @@
+use js_sys::RegExp as JSRegExp;
+
+use crate::common::{RegExpError, RegExpFlags};
+
+pub struct RegExp {
+    inner: JSRegExp,
+}
+
+impl RegExp {
+    pub fn new(pattern: &str, flags: Vec<RegExpFlags>) -> Result<Self, RegExpError> {
+        let mut flags_as_str: String = flags.into_iter().map(|flag| String::from(flag)).collect();
```

Comment:
fwiw I'd just reuse the `flags` name, the previous value is not available anyway because you consume it